blockchain/blockchain-wallet-v4-frontend
# fix(webpack-dev-server): Fix CSP nonce error in console.

Replace `**CSP_NONCE**` in index.html using the webpack-dev-server (as is done in production).

## Description (optional)

Finally fix the error message that always comes up first in the console when running locally:

`Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval'". Either the 'unsafe-inline' keyword, a hash ('sha256-yZmdgVvzROiMMhereftUxrofbQsrGpeQmdlyusoGIp4='), or a nonce ('nonce-...') is required to enable inline execution.`

## Testing Steps (optional)

Start the application locally and open the JavaScript console.  Notice the absence of the above error.
